docs/outputs.md

```diff
@@ -96,6 +96,29 @@ To enable this, add the setting `output_paf: True`.   The alignment will
 be written to files named `sim.hapA.paf` and `sim.hapB.paf`, representing the alignment
 to the reference of `sim.hapA.fa` and `sim.hapB.fa`, respectively.
 
+## Output novel adjacencies file
+A BED file containing the adjacencies introduced in the sequence by the simulated SVs.
+To enable, add the setting `output_adjacencies: True`.
+To connect the novel adjacencies to the grammar, we use the symbols of the grammar with a `+` if the novel adjacency 
+affects the end of the sequence, `-` for the beginning.
+Dispersions are disambiguated with a number representing their position.
+A novel adjacency with the end of the sequence just before the SV is denoted with PREV+ and NEXT- for the beginning 
```

Comment:
nit: use shorter notation (e.g. "p"/"s" or "s"/"e") for the flanking reference segments